test/AppKernel.php

```diff
@@ -11,12 +11,18 @@ class AppKernel extends Kernel
     private $configuration;
     private $enabledPlugins;
 
-    public function __construct(array $configuration, array $enabledPlugins)
+    private $cacheDir;
+    private $logsDir;
+
+    public function __construct(array $configuration, array $enabledPlugins, $uniqueIdentifier, $cacheDir, $logsDir)
     {
-        parent::__construct('test', true);
+        parent::__construct('test' . $uniqueIdentifier, true);
```

Comment:
Without PHPUnit's `--process-isolation`, it breaks on the following `Fatal error: Cannot redeclare class testTestDebugProjectContainer in /private/var/folders/51/zjs4tmks4kz7t5lt6wy6t7n00000gn/T/symfony-phpunit559a2f67cd539/cache/testTestDebugProjectContainer.php on line 1012`